pkg/sdk/testint/security_integrations_gen_integration_test.go

```diff
@@ -0,0 +1,380 @@
+					WithSaml2SnowflakeIssuerUrl(&issuerURL).
+					WithSaml2SpInitiatedLoginPageLabel(sdk.Pointer("label")).
+					WithAllowedEmailPatterns([]sdk.EmailPattern{{Pattern: "^(.+dev)@example.com$"}}).
+					WithAllowedUserDomains([]sdk.UserDomain{{Domain: "example.com"}}),
+			)
+		err := client.SecurityIntegrations.AlterSAML2Integration(ctx, setRequest)
+		require.NoError(t, err)
+
+		details, err := client.SecurityIntegrations.Describe(ctx, id)
+		require.NoError(t, err)
+
+		assertSAML2Describe(details, saml2details{
+			provider:                  "Custom",
+			enableSPInitiated:         "true",
+			spInitiatedLoginPageLabel: "label",
+			ssoURL:                    "https://example.com",
+			issuer:                    "test",
+			requestedNameIDFormat:     "urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified",
+			forceAuthn:                "true",
+			postLogoutRedirectUrl:     "http://example.com/logout",
+			signrequest:               "true",
+			comment:                   "a",
+			snowflakeIssuerURL:        issuerURL,
+			snowflakeAcsURL:           acsURL,
+			allowedUserDomains:        "[example.com]",
+			allowedEmailPatterns:      "[^(.+dev)@example.com$]",
+		})
+
+		unsetRequest := sdk.NewAlterSAML2IntegrationSecurityIntegrationRequest(id).
+			WithUnset(
+				sdk.NewSAML2IntegrationUnsetRequest().
+					WithSaml2ForceAuthn(sdk.Pointer(true)).
+					WithSaml2RequestedNameidFormat(sdk.Pointer(true)).
+					WithSaml2PostLogoutRedirectUrl(sdk.Pointer(true)),
+			)
+		err = client.SecurityIntegrations.AlterSAML2Integration(ctx, unsetRequest)
+		require.NoError(t, err)
+
+		details, err = client.SecurityIntegrations.Describe(ctx, id)
+		require.NoError(t, err)
+		assert.Contains(t, details, sdk.SecurityIntegrationProperty{Name: "SAML2_FORCE_AUTHN", Type: "Boolean", Value: "false", Default: "false"})
+		assert.Contains(t, details, sdk.SecurityIntegrationProperty{Name: "SAML2_REQUESTED_NAMEID_FORMAT", Type: "String", Value: "urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress", Default: "urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress"})
+		assert.Contains(t, details, sdk.SecurityIntegrationProperty{Name: "SAML2_POST_LOGOUT_REDIRECT_URL", Type: "String", Value: "", Default: ""})
+	})
+
+	t.Run("AlterSAML2Integration - REFRESH SAML2_SNOWFLAKE_PRIVATE_KEY", func(t *testing.T) {
+		id := testClientHelper().Ids.RandomAccountObjectIdentifier()
+		createSAML2Integration(t, id, nil)
+
+		setRequest := sdk.NewAlterSAML2IntegrationSecurityIntegrationRequest(id).WithRefreshSaml2SnowflakePrivateKey(sdk.Pointer(true))
+		err := client.SecurityIntegrations.AlterSAML2Integration(ctx, setRequest)
+		require.NoError(t, err)
+	})
+
+	t.Run("AlterSCIMIntegration", func(t *testing.T) {
+		id := testClientHelper().Ids.RandomAccountObjectIdentifier()
+		createSCIMIntegration(t, id, nil)
+
+		setRequest := sdk.NewAlterSCIMIntegrationSecurityIntegrationRequest(id).
+			WithSet(
+				sdk.NewSCIMIntegrationSetRequest().
+					WithEnabled(sdk.Bool(true)).
```

Comment:
Network policy is not here, as mentioned previously because of the quoting I think it would fail.

Reply:
Added.